cpp/unreal_plugins/UrdfBot/Source/UrdfBot.Build.cs

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+using UnrealBuildTool;
+using System;
+using System.Diagnostics;
+using System.IO;
+
+public class UrdfBot : ModuleRules
+{
+    public UrdfBot(ReadOnlyTargetRules Target) : base(Target)
+    {
+        PCHUsage = ModuleRules.PCHUsageMode.UseExplicitOrSharedPCHs;
+        bEnableExceptions = true;
+
+        PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "CoreUtils", "Engine", "InputCore", "XmlParser" });
+        PrivateDependencyModuleNames.AddRange(new string[] { "Slate" });
```

Comment:
Why is Slate needed, and why is it private?

Reply:
Slate is required to use custom keyboard control function such as `BindKey`. It can be public.

Reply:
Are you sure?
- `Slate` seems to be a UI framework for in-editor UI systems
  - https://docs.unrealengine.com/4.27/en-US/ProgrammingAndScripting/Slate/.
- `BindAxis` and `BindKey` seem to be part of the `Engine` module, and `EKeys` seems to be part of the `InputCore` module
  - https://docs.unrealengine.com/4.26/en-US/API/Runtime/Engine/Components/UInputComponent/BindAxis/
  - https://docs.unrealengine.com/4.26/en-US/API/Runtime/Engine/Components/UInputComponent/BindKey/
  - https://docs.unrealengine.com/4.26/en-US/API/Runtime/InputCore/EKeys/

Reply:
There is some compiling error when I remove Slate.

Reply:
Ok. Let's make it a public module dependency in that case. What is the error?